src/points/PointsHome.tsx

```diff
@@ -121,7 +122,12 @@ export default function PointsHome({ route, navigation }: Props) {
               />
             </View>
             <View style={styles.balanceRow}>
-              <Text style={styles.balance}>{pointsBalance}</Text>
+              <NumberTicker
+                testID="PointsBalance"
+                textStyles={styles.balance}
+                textHeight={48}
```

Comment:
1. What's this magic number? I had to drill down into `styles.balance` and then to `typeScale.displaySmall` to find out.
2. It will go out of sync if suddenly the `lineHeight` property in the applied style changes.
3. I guess it could be less painful to measure the height inside the component, instead of forcing the engineer to figure it out.

Reply:
i agree with your concerns but also they're kind of reasons for why i added this prop. for this particular case, the textHeight is equal to the lineHeight of the textStyles but they can also be independent. it is a prop that sets how tall the component is. by allowing the text styles to be customised, i think it becomes dangerous to derive the height based on the passed in styles (what if there are margins and paddings aside from the line height?)

i guess this complexity can be avoided by not allowing the styles to be customised for now? i thought i'd add it in while the animation logic is fresh in my mind, rather than punt it to the future when someone needs to figure it out

Reply:
I think it would make sense to have a single number affecting the size (either a prop or a style)

currently, I can provide any arbitrary number in this prop and get something I don't know why I would ever need:

| 100px | 20px |
|--------|--------|
| <img width="312" alt="image" src="https://github.com/valora-inc/wallet/assets/2737872/aa001b7e-97dc-4a4c-8236-fb493bdac5a5"> | <img width="312" alt="image" src="https://github.com/valora-inc/wallet/assets/2737872/1529a840-8fad-4a3a-a79e-9d062d168fd7"> |